src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -277,6 +279,13 @@ const formConfig = {
           uiSchema: unemployabilityFormIntro.uiSchema,
           schema: unemployabilityFormIntro.schema,
         },
+        uploadForm4192: {
+          title: 'Upload Form 4192',
+          path: 'new-disabilities/4192-upload',
+          depends: isUploading4192Form,
```

Comment:
I'm assuming right now we can't get to the page because the 4192 intro page isn't implemented yet?

Reply:
Correct.  This will be accessible when the prior page is added.